chi_analysis/plot_shapelimit_qbh.py

```diff
@@ -0,0 +1,150 @@
+from ROOT import *
```

Comment:
This script can also be dropped, since plot_shapelimit_13TeV can calculate the QBH as well.